### ISSUE TITLE
Bug in FeatureVector.normalizeToOne (RM3 results are wrong)

### ISSUE THREAD

**Issue by lintool** [lintool/Anserini#40]
We should be dividing by the length of the vector here:
https://github.com/lintool/Anserini/blob/master/src/main/java/io/anserini/rerank/rm3/FeatureVector.java#L167

Since RM3 uses this, RM3 results are probably wrong.

**Comment by lintool** [comment]
Indeed, RM3 effectiveness goes up after bug fix:

```
Tweets 2011
             MAP     P30
baseline   0.3532  0.3993
RM3        0.3877  0.4218  (before)
RM3        0.3939  0.4340  (fixed)

Tweets 2012
             MAP     P30
baseline   0.2090  0.3311
RM3        0.2361  0.3463  (before)
RM3        0.2420  0.3514  (fixed)

Tweets 2011+2012
             MAP     P30
baseline   0.2744  0.3620
RM3        0.3048  0.3806  (before)
RM3        0.3109  0.3889  (fixed)
```

**Comment by lintool** [comment]
Fixed commit af9cb2ae5a3515825a3d06b0c953967adfb908a9
